### DISCUSSION TITLE
Allow empty pages

### DISCUSSION THREAD

**Discussion by mjamro** [requarks/wiki#5463] [Help / Questions] [answered]
**Current behaviour:**

When you try to save an empty page you get a "Page content cannot be empty" error. 

**Proposed change**

When you save an empty page the page is saved/created correcty.

**Reason**

Empty pages are valid and suprisingly useful. You can use them to quickly prototype the layout and navigation flow of your wiki without any actual content and fill the content later at your convinience. 

As a reference you can create an empty text file or an empty MS Word Document without any trouble. There's nothing wrong with empty files and there is no reason to over-validate empty wiki pages.

**Comment by geez0r** [comment]
I'd say it's a non issue, just add "placeholder" or Lorem Ipsum and save it.

**Comment by NGPixel** [comment, marked as answer]
As @geez0r mentionned, simply add a dummy placeholder. The empty content guard is in place to prevent user errors.
